shield/internal/evaluator/evaluator_test.go

```diff
@@ -71,19 +79,21 @@ func TestEvalBooleanExpression(t *testing.T) {
 func TestArithmeticInteger(t *testing.T) {
 	tests := []struct {
 		input    string
-		expected int64
+		expected *big.Int
 	}{
-		{"1 + 2", 3},
-		{"10 - 4", 6},
-		{"2 * 5", 10},
-		{"8 / 2", 4},
-		{"2 + 2 * 2", 6},
-		{"2 * 2 + 2", 6},
-		{"(2 + 2) * 2", 8},
-		{"(10 + 2) / (6 - 3) * 4", 16},
-		{"5 / 2 * 2", 4},
-		{"-5 + 5", 0},
-		{"-(4 + 3) - (4 + 3)", -14},
+		{"1 + 2", big.NewInt(3)},
+		{"10 - 4", big.NewInt(6)},
+		{"2 * 5", big.NewInt(10)},
+		{"8 / 2", big.NewInt(4)},
+		{"2 + 2 * 2", big.NewInt(6)},
+		{"2 * 2 + 2", big.NewInt(6)},
+		{"(2 + 2) * 2", big.NewInt(8)},
+		{"(10 + 2) / (6 - 3) * 4", big.NewInt(16)},
+		{"5 / 2 * 2", big.NewInt(4)},
+		{"-5 + 5", big.NewInt(0)},
+		{"-(4 + 3) - (4 + 3)", big.NewInt(-14)},
+		{"(18446744073709551616 * 8 + 24) / 4 - 1", new(big.Int).SetBytes([]byte{0x02, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x05})},
```

Comment:
let's also add the simplest test case:
`9223372036854775807+1` must return `9223372036854775808`

(`9223372036854775807` is the maximum int64 value, in the current version it overflows to `-9223372036854775808` 😱)